src/Card.js

```diff
@@ -3,13 +3,33 @@ import CardDescription from './Card-Description';
 import CardStyle from './Card.css';
 
 export default class Card extends Component {
-    render() {
+  constructor() {
+    super();
+    this.state = {showInfo: false}
+  }
+
+  toggleInfo = (event) => {
+    this.setState({showInfo: !this.state.showInfo});
+    event.target.previousSibling.classList.toggle('expanded-description');
+    event.target.previousSibling.previousSibling.classList.toggle('expanded-img');
+    event.target.parentElement.classList.toggle('expanded-card');
+  }
+
+  render() {
+    const showInfo = this.state.showInfo;
+    let icon;
+    icon = showInfo ? ' - ' : ' + ';
```

Comment:
This can be merged with the line above it. `let icon = showInfo ? ' - ' : ' + ';`